Demo/ImagePickerDemo/ImagePickerDemo/Cell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Cell.swift
+//  ImagePickerDemo
+//
+//  Created by Khoa Pham on 11/8/15.
+//  Copyright © 2015 Ramon Gilabert Llop. All rights reserved.
+//
```

Comment:
@onmyway133 We usually remove those auto generated lines in our source code files.